### PR TITLE
Added first solution to problem 135

### PR DESCRIPTION
**Pull request by sharmapulkit04**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Review by poyea — APPROVED** [review]
Thank you for your pull request!🤩

File: project_euler/problem_135/sol1.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                common_difference /= 4
+                if (
+                    first_term > common_difference
+                    and first_term < 4 * common_difference
+                ):  # since x,y,z are positive integers
+                    frequency[n] += 1  # so z>0 and a>d ,also 4d<a
+    count = 0
+    for n in range(1, limit):
+        if frequency[n] == 10:
+            count += 1
```
Comment (poyea) [inline project_euler/problem_135/sol1.py:57]:
```suggestion
    count = sum(1 if x == 10 else 0 for x in frequency[1:limit])
```

Comment (sharmapulkit04) [reply project_euler/problem_135/sol1.py]: Done!

File: project_euler/problem_135/sol1.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                continue
+            else:
+                common_difference /= 4
+                if (
+                    first_term > common_difference
+                    and first_term < 4 * common_difference
+                ):  # since x,y,z are positive integers
+                    frequency[n] += 1  # so z>0 and a>d ,also 4d<a
+
+    count = sum(1 if x == 10 else 0 for x in frequency[1:limit])
```
Comment (dhruvmanila) [inline project_euler/problem_135/sol1.py:55]:
```suggestion
    count = sum(1 for x in frequency[1:limit] if x == 10)
```
No point in adding 0.

**Review by dhruvmanila — CHANGES_REQUESTED** [review]
One minor tweak and this is good to go!

File: project_euler/problem_135/sol1.py (hunk 2 above)
Comment (sharmapulkit04) [reply project_euler/problem_135/sol1.py]: Done!

**Review by dhruvmanila — APPROVED** [review]
LGTM! 🎉

**Pull request merged**